modules/CampingReservationClient.js

```diff
@@ -0,0 +1,64 @@
+const bent = require('bent')
+const Bottleneck = require('bottleneck')
+const TableBoi = require("./TableBoi");
+const reservationClient = bent('https://www.recreation.gov/api/camps/availability/campground/', 'json', 200)
+const limiter = new Bottleneck({
+  minTime: 1000 // 1 request per second max
+})
+
+/*
+Example api response:
+
+    "campsites":{
+          "100":{
+             "availabilities":{
+                "2020-10-01T00:00:00Z":"Reserved",
+                "2020-10-02T00:00:00Z":"Available",
+                ...
+             },
+             "site":"044"
+          },
+          ...
+      }
+
+*/
+
+/* returns map of sites to the available days for that site. Only available sites are returned.
+    ex: {
+      "044": [11, 12, 13],
+      "045": [13]
+    }
+*/
+async function getReservations(month) {
+  const campgroundId = "232447";  // currently hard-coded to Yosemite's Upper Pines
+  var response = await reservationClient(`${campgroundId}/month?start_date=2020-${month}-01T00%3A00%3A00.000Z`);
```

Comment:
can this be let?

Reply:
yup, will change

Reply:
is this going to break in 2021

Reply:
I'll change it to get the year dynamically